src/components/card-feature.js

```diff
@@ -36,6 +37,19 @@ export class Card {
       if (modelData.vscan && modelData.vscan.infected_files) {
         this.infected_files = modelData.vscan.infected_files;
       }
+
+      if (modelData.badges && modelData.badges.status) {
+        switch(modelData.badges.status.toLowerCase()) {
+          case 'active':
+            this.badge_status_image = '/img/active_flame_final_28w_35h.svg';
+            break;
+          case 'inactive':
+            this.badge_status_image = '/img/inactive_zzz_final_32w_35h.svg';
+            break;
+          default:
+            this.badge_status_image = null;
```

Comment:
This should be changed so that the default image is 'img/pending_review_final_29w_35h.svg'.
This image is to be used when a repo has not been reviewed yet.